scripts/py_matter_idl/matter/idl/tests/outputs/large_all_clusters_app/cpp-app/cluster-config/Descriptor.h

```diff
@@ -0,0 +1,48 @@
+// DO NOT EDIT - Generated file
+//
+// Application configuration for Descriptor
```

Comment:
Discussed offlline: this should mention it's based purely on ZAP data and is not necessarily the values used by the implementation.